src/deluge/model/voice/voice_sample_playback_guide.h

```diff
@@ -32,10 +32,12 @@ class VoiceSamplePlaybackGuide final : public SamplePlaybackGuide {
 	bool shouldObeyLoopEndPointNow();
 	int32_t getBytePosToStartPlayback(bool justLooped);
 	int32_t getBytePosToEndOrLoopPlayback();
-	LoopType getLoopingType(Source* source);
+	LoopType getLoopingType(const Source& source) const;
```

Comment:
I have a weak preference for east const rather than west as it makes the binding direction consistent, but I don't know if we ever actually agreed to a standard.

Reply:
oh I think we did for pointers but I forgot about it for references lol

Reply:
consensus (in dev discord) seems to be that at least for references, west const is more readable